pkgs/tools/backup/zfsbackup/deps.nix

```diff
@@ -1,273 +1,1326 @@
-# This file was generated by https://github.com/kamilchm/go2nix v1.3.0
+# file generated from go.mod using vgo2nix (https://github.com/adisbladis/vgo2nix)
```

Comment:
You should use https://github.com/nix-community/vgo2nix or buildGoModule.

Reply:
This file was indeed generated using `vgo2nix`.